dev/reference/alert-rules.md

```diff
@@ -618,6 +622,10 @@ For the critical-level alerts, a close watch on the abnormal metrics is required
 
     `sum(increase(tidb_tikvclient_gc_action_result{type="fail”}[1m])) > 10`
 
+    > **Note:**
+    >
+    > In TiDB 3.* versions, the `tidb_tikvclient_gc_action_result` metric exists but does not have a value. It's because garbage collection (GC) is introduced in TiDB 3.* versions but will not be performed in TiDB.
```

Comment:
```suggestion
    > In TiDB 3.* versions, the `tidb_tikvclient_gc_action_result` metric exists but does not have a value. It's because distributed garbage collection (GC) is introduced in the TiDB 3.0 version but will not be performed in TiDB.
```